lib/testmap.py

```diff
@@ -374,6 +374,24 @@ def tests_for_image(image: str) -> Sequence[str]:
     return list(tests)
 
 
+def tests_for_fnmatch(pattern: str) -> Sequence[str]:
+    """Return all contexts which match the given pattern."""
+
+    # Iterate all scenarios on all repositories, excluding _manual
+    tests = set()
+    for repo, branch_contexts in REPO_BRANCH_CONTEXT.items():
+        for branch, contexts in branch_contexts.items():
+            if branch.startswith('_'):
+                continue
+            for context in contexts:
+                c = context + '@' + repo
```

Comment:
like you say this probably isn't right. I think this should work on the target repository so when I do
```
./tests-trigger --repo cockpit-project/cockpit 22330 '*/networking'
```
I'd like to trigger all images that run cockpit tests and their `/networking` scenario